lang/java/avro/src/main/java/org/apache/avro/reflect/ReflectData.java

```diff
@@ -769,12 +770,13 @@ public Protocol getProtocol(Class iface) {
         iface.getPackage() == null ? "" : iface.getPackage().getName());
     Map<String, Schema> names = new LinkedHashMap<>();
     Map<String, Message> messages = protocol.getMessages();
+    Map<TypeVariable<?>, Type> genericTypeVariableMap = ReflectionUtil.resolveTypeVariables(iface);
     for (Method method : iface.getMethods())
```

Comment:
Not part of the PR, but can we add brackets here, looks awkward.

Reply:
Done

Reply:
Thanks!